client/components/tinymce/style.scss

```diff
@@ -761,8 +754,14 @@ div.mce-path {
 	margin: 2px 0 0 -13px #{"/*rtl:ignore*/"};
 	border: none;
 	color: $gray;
+
+	// Displaying the icon by using an svg file is a temporary solution to remove Noticons.
 	&::before {
-		@include noticon( '\f431' , 16px );
+		display: block;
+		content: url("/calypso/images/tinymce-icons/gridicons-chevron-down.svg");
+		width: 14px;
```

Comment:
The icon is small because I didn't want to be obtrusive. I didn't want to change padding or margins to accommodate a larger icon.